File: core/lib/Models/Warehouse.cs

namespace our.orders.Models
{
    public class Warehouse : Model
    {
        public string Name { get; set; }

        public string Address { get; set; }

        public string City { get; set; }

        public string CountryIso { get; set; }
        public string Email { get; set; }

        public string Phone { get; set; }

        public string PostalCode { get; set; }

        public string State { get; set; }

        public string VATNumber { get; set; }

        public override string Preview() => $"{Name}";
    }
}

File: core/app/src/@types/our-orders/Warehouse.ts
import { ModelBase } from '.';

export interface Warehouse extends ModelBase {

    Name: string;

    Address: string;

    City: string;

    CountryIso: string;

    Email: string;

    Phone: string;

    PostalCode: string;

    State: string;

    VATNumber: string;
}
